app/v2/notifications/get_notifications.py

```diff
@@ -83,7 +83,8 @@ def get_notifications():
         older_than=data.get('older_than'),
         client_reference=data.get('reference'),
         page_size=current_app.config.get('API_PAGE_SIZE'),
-        include_jobs=data.get('include_jobs')
+        include_jobs=data.get('include_jobs'),
+        count_pages=False
```

Comment:
```suggestion
        count=False
```

Reply:
Are you sure Leo? I think it should be `count_pages` - see https://github.com/alphagov/notifications-api/blob/master/app/dao/notifications_dao.py#L236